go/externals/services.go

```diff
@@ -72,14 +72,14 @@ func (p *proofServices) ListProofCheckers() []string {
 	return ret
 }
 
-func (p *proofServices) ListServicesThatAcceptNewProofs(mctx libkb.MetaContext) []string {
+func (p *proofServices) ListServicesThatAcceptNewProofs(mctx libkb.MetaContext) []keybase1.ListableProofService {
```

Comment:
could do the sorting here and dodge all the other changes

Reply:
![elderly-man-pondering-a-decision-GNMDC2](https://user-images.githubusercontent.com/1275828/56050233-4bdf4600-5d19-11e9-80a0-bbff9c0dc3a3.jpg)